### DISCUSSION TITLE
Explaination about `intercept_unknown`

### DISCUSSION THREAD

**Discussion by BobSquarePants** [evilsocket/opensnitch#1231] [Development]
Hi,

in [wiki:configuration](https://github.com/evilsocket/opensnitch/wiki/Configurations) we can read the follwing

> ... As of v1.4.0rc2 version, it's safe to set it to false, **and just let it drop those "unknown" connections.** It's up to you. Most of the connections intercepted by this option are those in a bad state or similar. ...

So is it mean that if `InterceptUnknown` is set to `True` that opensnitch will catch a drop them by default ? or will it follow the `DefaultAction` ?
and `False` opensnitch will not handle them.

Thanks

( when solved I'll make a PR )

**Comment by gustavo-iniguez-goya** [comment]
When `InterceptUnknown` is set to False, it'll apply the DefaultAction configured.
When it's set to True, it'll ask the user to allow/deny generic "Outbound connections", this is, a network connection not associated to a process.

This situation can occur in several scenarios:
  - When the kernel considers a network connection as NEW when it's not.
    The conntrack table may consider a connection as NEW not only the TCP SYN packets, but any packet that it sees for the first time.
    This can be a reply for example, or a FIN packet. In this scenario, the process may not exist when we intercept it (this is what we call "bad state" in the wiki).
  - Kernel connections.
